domains.txt

```diff
@@ -260,11 +295,13 @@ giostechnology.com
 glglobal.co.uk
 global-m.co.uk
 global-resourcing.com
+globalsearchgroup.co.uk
```

Comment:
At Global Search Group we specialise in full recruitment and staffing services both in the UK and Internationally.